client/src/components/HomePage/HowAddEvent.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Row, Col, Card, CardTitle } from 'react-materialize';
+
+
+const HowAddEvent = () => {
+  return (
+    <Row>
+      <Col s={4}>
+        <Card horizontal header={
+          <CardTitle
+            image="http://www.thebluediamondgallery.com/handwriting/images/register.jpg" />
```

Comment:
Please do not use links to external resources. Copy them to source tree and use local copies.